### DISCUSSION TITLE
How to set the link in the web button in the UI

### DISCUSSION THREAD

**Discussion by wocmultimedia** [leokhoa/laragon#647] [Q&A]
Hi,
I'm new to this and I just set Laragon for the second time. I followed the instructions to install Laragon then Wordpress and the site I called "myblog" . It created the pretty url myblog.test and it works. But if I start the Laragon interface and click on the web icon it directs me to another URL called localhost. 
I previously had a Bitnami installation that I uninstalled because conflicted with Laragon. Then I made a fresh installation of both Laragon and wordpress calling the project "myblog".
My question is how I can change the link in this web icon to point to my project "myblog"? I'm sorry but I didn't find any hint in the FAQ.
Thank you
![schermatalaragon](https://github.com/leokhoa/laragon/assets/2352240/826daef0-0ef9-41b5-80cd-37305a719782)
![image](https://github.com/leokhoa/laragon/assets/2352240/024a2570-4b10-4ee0-9df8-b20b05b595b9)

**Comment by jcvasquez07** [comment]
By default when you click on the 'Web' button it loads a file named 'index.php'. Just replace the original one with your own version, redirecting to your wordpress site (you can use the header() function for that)

**Reply by wocmultimedia** [reply]
[index.zip](https://github.com/leokhoa/laragon/files/13821526/index.zip)
Hi @jcvasquez07 ,
Where do I find this index.php? I checked in the laragon > www folder > index.php but I cannot understand what I have to do
there's no reference to locallhost.
This is the index-php. Please, let me know if it's the one you mentioned and what I have to change [index.zip](https://github.com/leokhoa/laragon/files/13821526/index.zip)
Thanks a lot

**Reply by jcvasquez07** [reply]
Just rename the file to simething like 'index.php.bak', create a new one and write just this:
```
<?php
header("location:path/to/my/wordpress/index.php");
```
Just write the correct path to your WP installation

**Reply by wocmultimedia** [reply]
I made a copy of the  index.php which is in MyDrive:\laragon\www\ then I modified it following your suggestion but I don't get nothing else than a blank page with address http://localhost 
Anyhow my wordpress index.php is located in  MyDrive:\laragon\www\myblog\  can you write the full code because I'm not a developer. Thanks

**Comment by jcvasquez07** [comment]
If the entry point of your WP installation is the file "
MyDrive:\laragon\www\myblog\index.php" (sorry, not familiar with WP, never
used it),
then your file MyDrive:\laragon\www\ index.php should be:

<?php
header("location: myblog\index.php");


<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Libre
de virus.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Reply by wocmultimedia** [reply]
Hi, Hola Juan Carlos,

I'm sorry but it doesn't work whatever change I make it gave only "not found".

There's no problem I have the tab opened so I can reach the site or from the menu I go to www > myblog and it open the site.

Thanks.
